src/ctia/entity/feed.clj

```diff
@@ -139,7 +140,7 @@
           (fn [feeds]
             (map #(decrypt-feed % services) feeds))))
 
-(def fetch-limit 200)
+(def fetch-limit 10000)
```

Comment:
The ElasticSearch documentation recommends to not request too many results at once as it can significantly increase memory and CPU usage. If I'm not wrong, the `fetch-limit` set the page size used in the function `list-all-pages`. All pages are always fetched. I don't think you need to change this value to retrieve all judgements.

Reply:
I second Matt, you can raise it to limit the number of queries but it will retrieve all the relationships (which is potentially dangerous, we had trouble with it).

Reply:
OK I will keep the current value